### ISSUE TITLE
[design] Size of Nextcloud Folder below Folder Path only 50% visible

### ISSUE THREAD

**Issue by mr-bolle** [nextcloud/talk-android#832]
### Steps to reproduce
Start Talk Android App
Select a User / Group
Add any Attachment 

### Expected behaviour

Reduce the Font Size, that the Folder Name and the additional information are full visible.

:bulb: it would be great if it will be posible to have a sorting filter (Name / Age)

### Actual behaviour
Below the Folder Path is visible the Size of the Folder, and the last changed Date.
This line is only 50% visible.

<img src="https://user-images.githubusercontent.com/22414212/81479373-14481380-9223-11ea-9f1c-a1068b2dee94.jpg" width="60%">

## Server configuration detail

**Operating system:** Linux 4.9.0-12-amd64 #1 SMP Debian 4.9.210-1 (2020-01-20) x86_64

**Webserver:** Apache/2.4.38 (Debian) (apache2handler)

**Database:** mysql 10.4.12

**PHP version:** 

7.3.17
Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, ctype, curl, dom, fileinfo, filter, ftp, hash, iconv, json, mbstring, SPL, PDO, session, posix, Reflection, standard, SimpleXML, pdo_sqlite, Phar, tokenizer, xml, xmlreader, xmlwriter, mysqlnd, apache2handler, apcu, exif, gd, gmp, imagick, intl, ldap, memcached, pcntl, pdo_mysql, pdo_pgsql, redis, sodium, zip, Zend OPcache

**Nextcloud version:** 18.0.4 - 18.0.4.2

**Updated from an older Nextcloud/ownCloud or fresh install:** nextcloud/docker 

**Where did you install Nextcloud from:** https://github.com/nextcloud/docker

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.4.0
 - activity: 2.11.0
 - calendar: 2.0.3
 - cloud_federation_api: 1.1.0
 - comments: 1.8.0
 - contacts: 3.3.0
 - dav: 1.14.0
 - deck: 1.0.0
 - documentserver_community: 0.1.5
 - federatedfilesharing: 1.8.0
 - federation: 1.8.0
 - files: 1.13.1
 - files_external: 1.9.0
 - files_pdfviewer: 1.7.0
 - files_rightclick: 0.15.2
 - files_sharing: 1.10.1
 - files_trashbin: 1.8.0
 - files_versions: 1.11.0
 - files_videoplayer: 1.7.0
 - firstrunwizard: 2.7.0
 - issuetemplate: 0.6.0
 - logreader: 2.3.0
 - lookup_server_connector: 1.6.0
 - mail: 1.3.4
 - music: 0.14.0
 - nextcloud_announcements: 1.7.0
 - notifications: 2.6.0
 - oauth2: 1.6.0
 - ocr: 6.0.37
 - onlyoffice: 4.1.4
 - password_policy: 1.8.0
 - photos: 1.0.0
 - privacy: 1.2.0
 - provisioning_api: 1.8.0
 - recommendations: 0.6.0
 - serverinfo: 1.8.0
 - settings: 1.0.0
 - sharebymail: 1.8.0
 - spreed: 8.0.8
 - support: 1.1.0
 - survey_client: 1.6.0
 - systemtags: 1.8.0
 - tasks: 0.12.2
 - text: 2.0.0
 - theming: 1.9.0
 - twofactor_backupcodes: 1.7.0
 - twofactor_totp: 4.1.3
 - updatenotification: 1.8.0
 - viewer: 1.2.0
 - workflowengine: 2.0.0
Disabled:
 - admin_audit
 - audioplayer
 - encryption
 - user_ldap
 - workflow_script

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "htaccess.RewriteBase": "\/",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "apps_paths": [
        {
            "path": "\/var\/www\/html\/apps",
            "url": "\/apps",
            "writable": false
        },
        {
            "path": "\/var\/www\/html\/custom_apps",
            "url": "\/custom_apps",
            "writable": true
        }
    ],
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "example.nextcloud.com"
    ],
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "18.0.4.2",
    "overwrite.cli.url": "http:\/\/example.nextcloud.com",
    "overwriteprotocol": "https",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "maintenance": false,
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpsecure": "tls",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_sendmailmode": "smtp",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "force_language": "de",
    "force_locale": "de_DE",
    "theme": "",
    "loglevel": 2
}
```
</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** 

**Operating system:** 

## Logs

<details><summary>Web server error log</summary>

```

```
</details>

<details><summary>Nextcloud log</summary>

```

```
</details>

<details><summary>Browser log</summary>

</details>

**Comment by mario** [comment]
I have never seen this. What phone and what Android version?

**Comment by mr-bolle** [comment]
Samsung Note 9 - SM-N960F
Android 10 - Stock ROM

**Comment by mario** [comment]
Ok, will hopefully be fixed in v9.0beta5.

**Comment by mr-bolle** [comment]
I installed the v9.0beta5 and the problem is solved.

![tempFileForShare_20200606-122952](https://user-images.githubusercontent.com/22414212/83942198-b8f14d00-a7f1-11ea-8cf9-89bc392a1e84.jpg)
